DestructionProcessing.cs

```diff
@@ -471,6 +471,8 @@ internal static void HandleVehicleCap()
                 {
                     if (vehicle.isDead)
                         continue;
+                    if (vehicle.passengers != null)
```

Comment:
To be honest, it was more of a suggestion than an implementation. But if you can add something like this, it would be nice.